### DISCUSSION TITLE
Irreproduceble example: 04a_1d_lgcp.R

### DISCUSSION THREAD

**Discussion by rondon-d** [inlabru-org/inlabru#214] [Q&A] [answered]
Hello team. 
I am running and reading the examples provided on the [website](https://sites.google.com/inlabru.org/inlabru/tutorials/practicals).
I am getting the following warning, and thus getting different results from the HTML document. 

pred_spde_lp <- predict(fita.spde, predf, ~ spde1D + Intercept)
Warning in handle_problems(e_input) :
  The input evaluation 'Intercept' for 'Intercept' failed. Perhaps the data object doesn't contain the needed variables? Falling back to '1'.

 I should get this: 
![image](https://github.com/inlabru-org/inlabru/assets/145334565/7a28cbef-0364-4d11-bebd-e1871266cfc8)

but I am getting this: 
![image](https://github.com/inlabru-org/inlabru/assets/145334565/245aaea4-4676-4cbf-be61-f11e77762ebb)

I am using INLA version ‘23.9.9’
and inlabru version ‘2.9.0‘

Thank you for your time

**Comment by finnlindgren** [comment, marked as answer]
The inla.org examples are outdated. I thought we had updated the front page text, but it seems it needs more clear text. The up-to-date examples are on https://inlabru-org.github.io/inlabru/

"The inlabru.org website has links to old tutorials with code examples for versions up to 2.1.13. For later versions, updated versions of these tutorials, as well as new examples, can be found at https://inlabru-org.github.io/inlabru/articles/"
